Reports/RSP_STOCKLEDGER.txt

```diff
@@ -76,35 +78,37 @@ SELECT * INTO #RESULTTABLE FROM
 			LEFT JOIN DIVISION D2 ON A.BILLTOADD = D2.INITIAL
 		) M 
 		INNER JOIN RMD_TRNPROD N ON M.VCHRNO=N.VCHRNO
+		LEFT JOIN vwVehicleBarcode VW ON N.BC=VW.BARCODE AND N.MCODE=VW.MCODE
```

Comment:
Use `BARCODE_DETAIL`  table.
Joining directly to transaction table will degrade SP Performance. 
Please optimize